src/main/webapp/js/adminHome.js

```diff
@@ -60,6 +60,37 @@ function enableAddInstructorForm() {
     });
 }
 
+function addInstructorAjax(isError, data) {
+    var rowText;
+    if (isError) {
+        rowText = createRowForResultTable('-', '-', '-', '-', false, 'Cannot send Ajax Request!');
+    } else {
+        rowText = createRowForResultTable(
+            data.instructorShortName,
+            data.instructorName,
+            data.instructorEmail,
+            data.instructorInstitution,
+            data.instructorAddingResultForAjax,
+            data.statusForAjax
+        );
+    }
+    $('#addInstructorResultTable tbody').append(rowText);
+    var isNotAddingResultForAjax = !(data && data.instructorAddingResultForAjax);
+    if (isInputFromFirstPanel && isNotAddingResultForAjax) {
```

Comment:
This doesn't look equivalent to before, before if it was errornous, isInputFromFirstPanel is enough, does this affect the functionality?